content/webapp/app.ts

```diff
@@ -118,6 +118,13 @@ const appPromise = nextApp
     route('/newsletter', '/newsletter', router, nextApp);
 
     route('/guides', '/guides', router, nextApp);
+    route('/guides/exhibitions', '/exhibition-guides', router, nextApp);
+    route(
+      `/guides/exhibitions/:id(${prismicId})/:type?`,
+      '/exhibition-guide',
+      router,
+      nextApp
+    ); // :type(${guideType})
```

Comment:
Did you decide against having a separate template for the `/:type?` route?

Reply:
it's still up for grabs, this is just a first pass